src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesList/FeatureStrategyCard/FeatureStrategyCard.styles.ts

```diff
@@ -43,6 +43,12 @@ export const useStyles = makeStyles(theme => ({
     icon: {
         fill: theme.palette.primary.main,
     },
+    disabledButton: {
+        color: '#585858 !important',
```

Comment:
Why important?

Reply:
Yes indeed it is important.

Reply:
There are usually better ways to do this than using `!important!`, because it can lead to unexpected behavior.

Reply:
I am not sure, but I think if I use `.Mui-disabled { background-color: blue; }` . it will override all the styles for disabled button. So using important is like an exception for this case. If I remove the important nothing happens.

Reply:
I think you just need to override the disabled button styles: https://codesandbox.io/s/l9pk2wl69q

What is this supposed to be doing by the way? Because I believe that if we "fix" disabled style for one kind of IconButton we should do it for all of them.

Reply:
I see. I think now it should be good. it is for the icons disabled in the side panel for strategies.

Reply:
Sorry for the miscommunication. These should not have any shadow after discussing with Elise we've elected to keep them as they were.